akka-distributed-data/src/main/scala/akka/cluster/ddata/PruningState.scala

```diff
@@ -19,10 +19,12 @@ import akka.util.unused
       if (seen(node) || owner.address == node) this
       else copy(seen = seen + node)
     }
+    def estimatedSize: Int = 58 + 50 * seen.size
```

Comment:
From where is 58 and 50 coming? Can we document it and/or put it on a constant?

Reply:
moved to constants, so now it should be self explaining: c442101